src/test/resources/net/sf/jabref/importer/fileformat/MedlineImporterTestSingleEntry.bib

```diff
@@ -1,15 +1,25 @@
-% Encoding: UTF-8
-
-@Article{,
```

Comment:
Why was the complete test case changed? Can't the old test case be kept and an additional test case be added?